dbt_project.yml

```diff
@@ -426,6 +426,12 @@ models:
       bnb:
         +schema: pancakeswap_bnb
         +materialized: view
+    bancor:
+      +schema: bancor
+      +materialized: view
+      bnb:
+        +schema: bancor_ethereum
+        +materialized: view
```

Comment:
You may mistype the blockchain name here.
```suggestion
    bancor:
      +schema: bancor
      +materialized: view
      ethereum:
        +schema: bancor_ethereum
        +materialized: view
```

Reply:
Got it. Thanks